content/applications/finance/accounting/customer_invoices/electronic_invoicing.rst

```diff
@@ -126,79 +126,100 @@ Registration
+     available.
 
    .. image:: electronic_invoicing/peppol-system-parameter.png
       :alt: Peppol test mode parameter
 
-   .. image:: electronic_invoicing/peppol-test-mode-settings.png
-      :alt: Peppol test mode selection
-
-Now, you can request a verification code to be sent to you by clicking :guilabel:`Verify phone
-number`.
-
-.. image:: electronic_invoicing/peppol-registration-verify.png
-   :alt: phone validation request verification
+.. seealso::
+   - `Peppol EAS - European Commision <https://ec.europa.eu/digital-building-blocks/wikis/display/DIGITAL/Code+lists/>`_
+   - `Peppol Endpoint - OpenPeppol eDEC Code Lists <https://docs.peppol.eu/edelivery/codelists/>`_
+     (open the "Participant Identifier Schemes" as HTML page)
 
-A text message containing a code is sent to the phone number provided to finalize the verification
-process.
+When set up, request a verification code to be sent to you by clicking
+:guilabel:`Send a registration code by SMS`. A text message containing a code is sent to the phone
+number provided to finalize the verification process.
 
-.. image:: electronic_invoicing/phone-registration.png
+.. image:: electronic_invoicing/peppol-phone-verification.png
    :alt: phone validation
 
-Once you enter the code and click :guilabel:`Confirm`, you can see that your registration is pending
-activation. From this point onwards, the default journal for receiving Vendor Bills can be set.
-
-.. image:: electronic_invoicing/peppol-registration-pending.png
-   :alt: pending application
+Once you enter the code and click :guilabel:`Register`, your Peppol participant status is updated.
+If you chose to only send documents, then the status changes to **Can send but not receive**.
+If you opted to receive as well, the status changes to **Can send, pending registration to
+receive**. In that case, it should be automatically activated within a day.
 
-It should be automatically activated within a day.
+From this point onwards, the default journal for receiving vendor bills can be set.
 
 .. tip::
    It is also possible to manually trigger the cron that checks the registration status by going to
    :menuselection:`Settings --> Technical --> Scheduled Actions --> PEPPOL\: update participant
    status`.
 
-Your application status should be updated soon after you are registered on the Peppol network.
+Your receiver application status should be updated soon after you are registered on the Peppol
+network.
 
-.. image:: electronic_invoicing/peppol-registration-active.png
-   :alt: active application
+.. image:: electronic_invoicing/peppol-receiver.png
+   :alt: receiver application
 
 All invoices and vendor bills can now be sent directly using the Peppol network.
 
+.. important::
+   If you need to update the email that our team can use to contact you, you need to click
+   :guilabel:`Update contact details` for the updated data to be sent to us.
```

Comment:
I wanted to emphasize that nothing would happen unless they clicked that button because the data needs to be updated on our server. But if you think it's clear to users like that, I don't mind.